core/modules/css/cssLib.i

```diff
@@ -47,7 +53,118 @@ namespace std {
     %template(StringVector) vector<string>;
 };
 
+%include "sql/SqlErrorObject.h"
+
+%{
+    void setPythonException(const std::string& exceptionStr, const lsst::qserv::css::CssError& ex) {
+        PyObject* module = PyImport_ImportModule("lsst.qserv.css");
+        if (!module)
+            throw;
+        PyObject* exception = PyObject_GetAttrString(module, exceptionStr.c_str());
+        if (not exception)
+            throw;
+        PyObject* what = PyString_FromString(ex.what());
```

Comment:
I'd replace this line and the rest of this function with one line:

    PyErr_SetString(exception, ex.what());